### PR TITLE
Adjust firewall_enabled test for SLE15

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/25442

### CODE REVIEW

**Comment by okurz** [comment]
@yxususe before I saw you picked up the ticket this is what I prepared, would you like to review and test?

**Comment by yxususe** [comment]
LGTM. But I'm afraid I cannot test it today, because I'm testing something else and can't always switch branches. However I see the problem that SuSEfilewall2 is extensively used in our code, which means we need to do quite some work to adapt the tests to SLE15.

File: tests/console/firewall_enabled.pm [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+use base 'opensusebasetest';
 use strict;
 use testapi;
 use utils;
 
 sub run {
-    assert_script_run("SuSEfirewall2 status");
-    if (is_jeos) {
+    if ((is_sle && sle_version_at_least('15')) || (is_leap && is_leap_version_at_least('15'))) {
+        assert_script_run('firewallctl state');
```
Comment (sysrich) [inline tests/console/firewall_enabled.pm]: I think we want to test firewallctl on Tumbleweed also

**Comment by okurz** [comment]
Yes, I checked TW. It still has the SuSEfirewall2 command installed by default and firewalld is available for install bit not fully usable.
The testing goal of the test module should be "ensure the firewall runs by default using the standard tool"

How about another explicit test module?

**Comment by okurz** [comment]
ok, I'm going with @yxususe's LGTM here and merge this fix for SLE15. I am pretty sure that there are many missing parts regarding firewallctl which need to be implemented anyway. I recommend therefore in this case to go forward with the changes for SLE15 as we need this fix anyway and then afterwards see about TW which is still running fine so a change is not necessary there right now.

**Comment by andreasstieger** [comment]
This breaks openSUSE Leap maintenance tests:

`# Test died: Undefined subroutine &firewall_enabled::is_leap_version_at_least called at /var/lib/openqa/cache/openqa1-opensuse/tests/opensuse/tests/console/firewall_enabled.pm line 21.
`
https://openqa.opensuse.org/tests/490634#step/firewall_enabled/2
https://openqa.opensuse.org/tests/490637#step/firewall_enabled/2

**Comment by okurz** [comment]
Sorry about that. Fixed in 7f870961, I will retrigger the tests

**Pull request merged**